front/src/styles/scss/applications/operationalStudies/_scenario.scss

```diff
@@ -144,10 +144,17 @@
           color: white;
           background-color: var(--secondary);
         }
+        &:hover {
+          background-color: var(--secondary);
+          color: var(--white);
+          .scenario-details-modify-button-text {
+            opacity: 1;
+          }
+        }
```

Comment:
Don't Repeat Yourself! You've already wrote this below, please try to use common classes.